### ISSUE TITLE
Specifying channel not working

### ISSUE THREAD

**Issue by molteninjabob** [jackcoded/newman-reporter-slackmsg#5]
I tried using the new channel option added but can't seem to get it working. Nothing is posting to the channel. The app is installed in our workspace and added to the channel I'm trying to post to. This is the command that I am running:

`newman run https://api.getpostman.com/collections/10839446-8485b6f7-f984-4889-bc27-cc60eb218c0b\?apikey=<postman-apikey> --environment https://api.getpostman.com/environments/10839446-3cf10fb8-b07b-4703-ad19-1a07069aaa45\?apikey=<postman-apikey> --folder <foldername> --suppress-exit-code -r slackmsg --reporter-slackmsg-webhookurl https://slack.com/api/chat.postMessage --reporter-slackmsg-token <oAuth token> --reporter-slackmsg-channel #postman-workflow-notifications`

I've tried variations on the channel argument - with and without the # sign, wrapping in single quotes and double quotes. Nothing is appearing in slack.

Here are the permissions given to the slackbot:
![Slack API: Applications | Weave Slack 2020-10-08 16-30-29](https://user-images.githubusercontent.com/11509961/95520054-af384b80-0983-11eb-80ac-d89dc33d58f9.jpg)

**Comment by jackcoded** [comment]
Your --reporter-slackmsg-token <oAuth token> is meant to be an actual token from here
![image](https://user-images.githubusercontent.com/22386251/95536378-abb1cc80-0a36-11eb-9e2d-7241f1305e11.png)

**Comment by molteninjabob** [comment]
Yes, I am putting in the actual token when I execute. I just didn't want to post the token here. And, I'm trying to post to channels the bot has already been added to. So, joining the channel isn't the issue. It just isn't posting.

**Comment by molteninjabob** [comment]
I pulled the code down locally and ran it and it worked. But, in our CI/CD pipeline, we're running these from Github Actions and that seems to be where the problem is. The webhook worked just fine, but for some reason, it won't post through the Postman API. I don't think it's an issue with your code, so I'll continue investigating on the Github side.
